tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/issue304_pytorch_gemv.py

```
@@ -0,0 +1,304 @@
+    print_header()
+    for shape in shapes:
+        weights, queries = make_tensors(torch, shape, torch.device("cpu"))
+        inner = inner_iterations(shape)
+        with torch.no_grad():
+            times = measure_wall(
+                torch,
+                backend=None,
+                runs=runs,
+                warmup=warmup,
+                inner=inner,
+                action=lambda run: torch.matmul(weights, queries[run % len(queries)]),
```

Ruff (0.15.12)

[warning] Function definition does not bind loop variable `weights`

(B023)